### PR TITLE
Add MediaStreamTrack Content Hints IDL file + test

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
These tests pass on Chrome Dev with experimental web platform features enabled.

File: mst-content-hint/idlharness.window.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+idl_test(
+  ['mst-content-hint'],
+  ['mediacapture-streams', 'dom'],
+  async idl_array => {
+    idl_array.add_objects({
+      MediaStreamTrack: ['audioTrack', 'videoTrack'],
+    });
+
+    const stream = await getNoiseStream({ audio: true, video: true });
+    self.audioTrack = stream.getAudioTracks()[0];
```
Comment (lukebjerring) [inline mst-content-hint/idlharness.window.js]: Nit: Pull `stream.getAudioTracks()[0]` into a variable.

File: mst-content-hint/idlharness.window.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    self.videoTrack = stream.getAudioTracks()[0];
```
Comment (lukebjerring) [inline mst-content-hint/idlharness.window.js]: The above nit assumes it was intended to re-use the same value. Is it? Or is there a getVideoTracks equivalent?

**Review by lukebjerring — APPROVED** [review]

File: mst-content-hint/idlharness.window.js (hunk 1 above)
Comment (foolip) [reply mst-content-hint/idlharness.window.js]: But it has to be on the global object for `add_objects` to work, I presume using `eval()`. Am I misunderstanding?

File: mst-content-hint/idlharness.window.js (hunk 2 above)
Comment (foolip) [reply mst-content-hint/idlharness.window.js]: Oops, copypasta, will fix.

File: mst-content-hint/idlharness.window.js (hunk 1 above)
Comment (lukebjerring) [reply mst-content-hint/idlharness.window.js]: DW - Irrelevant given that there should have been 2 different values :)

**Review by alvestrand — APPROVED** [review]

**Pull request merged**